docs/docs/resources/pipeline.yaml

```diff
@@ -0,0 +1,334 @@
+# `pipeline.yaml` contains the configurations for all components to be deployed
+#
+# In order to avoid repetition (for example with `namespace`), the user can set
+# values in `defaults.yaml` for all apps of a certain type (could be specific
+# like `producer` or broad like `kubernetes-app`). ]
+
+# Base Kubernetes App
+- type: kubernetes-app
+  name: kubernetes-app # required
```

Comment:
name is actually not required. I think it is set to type by default. Only if you have two components with same type in a pipeline, you must override it

Reply:
Interesting, the pydantic-generated schema says [name is required](https://github.com/bakdata/kpops/blob/main/schema_pipeline.json#L261) and it [doesn't seem to have a default value](https://github.com/bakdata/kpops/blob/main/schema_pipeline.json#L222).

I suppose the schema cannot contain a default that is not hard-coded?

Reply:
`name` is required

Reply:
Ah, we just configured it in our defaults to be automatically type if not set. Sorry for the confusion